collector/linux_threads_test.go

```diff
@@ -0,0 +1,27 @@
+package collector
```

Comment:
This file needs the copyright header, and probably `// +build !nothreads` to avoid test failures.